src/Auth/Login.tsx

```diff
@@ -30,30 +37,44 @@ const Login = () => {
     }
   };
 
-  const handleSubmit = (e: FormEvent) => {
+  const handleSubmit = async (e: FormEvent) => {
     e.preventDefault();
-    validateEmail(formValues.email);
-    validatePassword(formValues.password);
-
-    if (!errors.email && !errors.password) {
-      window.location.href = "/items";
-    }
+    const isFormValid = Object.values(errors).every((error) => !error);
+    if (isFormValid) {
+        try{
+          const result=await postSignIn(formValues);
+          const {accessToken}=result;
```

Comment:
여기에서는 result가 특정적인 의미 없이 await 요청의 결과값을 의미하고 있어서 아래와 같이 써도 좋을 것 같아요.
```suggestion
          const { accessToken }=await postSignIn(formValues);
```